qtpy/__init__.py

```diff
@@ -241,6 +241,19 @@ class PythonQtWarning(Warning):
 except (ImportError, PythonQtError):
     pass
 
+PYQT_VERSION_MIN = '5.9.0'
+PYSIDE_VERSION_MIN = '5.12.0'
+QT_VERSION_MIN = '5.9.0'
```

Comment:
Maybe this constants should go above? Near the `LEGACY_APIS` constant

Reply:
Yeah, I was going to move them but forgot, thanks. Doing that now.

Reply:
Done, also added a comment and added a second line break around the new private function